admin_pages/payments/templates/payment_log_details.template.php

```diff
@@ -31,34 +31,40 @@
             <tr>
                 <th>
                     <label>
-                        <?php _e('Payment Method', 'event_espresso'); ?>
+                        <?php esc_html_e('Payment Method', 'event_espresso'); ?>
                     </label>
                 </th>
                 <td>
-                    <?php echo $payment_method
+                    <?php
+                    if ($payment_log->object() instanceof EE_Transaction) {
+                        esc_html_e('Unknown', 'event_espresso');
```

Comment:
I think it'd be better here to get the payment method from the transation and then use that payment method's `admin_name()`?

Reply:
This is happening for the same reason as above, if the Transaction object is passed to the log call, we can't be sure that the last payment method on a transaction is the 'correct' payment method.